components/camel-jaxb/src/main/java/org/apache/camel/converter/jaxb/JaxbDataFormat.java

```diff
@@ -82,6 +82,7 @@
     private String contextPath;
     private String schema;
     private String schemaLocation;
+    private String noNamespaceSchemaLocation;
```

Comment:
You need a little more. The option should also be added to  `org.apache.camel.model.dataformat.JaxbDataFormat`